pkgs/tools/misc/colord/default.nix

```diff
@@ -60,6 +62,8 @@ stdenv.mkDerivation rec {
     "-Dvapi=true"
     "-Ddaemon=${lib.boolToString enableDaemon}"
     "-Ddaemon_user=colord"
+    (lib.mesonBool "systemd" enableSystemd)
+    (lib.mesonBool "udev_rules" (lib.elem "udev" udev.meta.pkgConfigModules))
```

Comment:
Ooh I don't like this

Reply:
Not all udev implementations (libudev-zero) implement udev rules, just the library interface.  To indicate that, they don't provide udev.pc, which is a file that only tells you the udev rules directory.  In that case, udev_rules must be disabled, or the build will fail due to it not being able to tell where to install its rules due to lack of udev.pc.  You might think that we could override that using pkg-config environment variables, but unfortunately it only looks at those _after_ it's found the .pc files.

Reply:
Wow this is cursed.